docker/docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  trin:
+    hostname: trin
```

I would add a `container_name` for more clarity.

```suggestion
  trin:
    container_name: trin
    hostname: trin
```